tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -3597,8 +3532,62 @@ def test_extra_nested_branches_exception_logged(self, setup):
         ):
             self.checker.visit_try(try_node)
 
+
+class TestCheckDoNotUseLegacyTyping(pylint.testutils.CheckerTestCase):
```

Comment:
can we remove this looks like it got added in a merge :)

Reply:
It has been removed.